examples/ml-fraud-detection/akka-streams/src/main/scala/cloudflow/examples/frauddetection/ingress/GenerateTransactions.scala

```diff
@@ -0,0 +1,116 @@
+package cloudflow.examples.frauddetection.ingress
+
+import java.util.UUID
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import cloudflow.akkastream.AkkaStreamlet
+import cloudflow.akkastream.scaladsl.RunnableGraphStreamletLogic
+import cloudflow.examples.frauddetection.data.CustomerTransaction
+import cloudflow.streamlets.{ IntegerConfigParameter, StreamletShape }
+import cloudflow.streamlets.avro.AvroOutlet
+import org.joda.time.{ DateTime, DateTimeZone }
+import cloudflowx.config.ConfigUtil
+import cloudflowx.config.ConfigUtil.implicits._
+import cloudflowx.ingress.RecordsReader
+
+import scala.concurrent.duration._
+
+/**
+ * Reads transaction records from a CSV file,
+ * parses it into a CustomerTransaction and sends it downstream.
+ */
+class GenerateTransactions extends AkkaStreamlet {
+
+  //\\//\\//\\ INLETS //\\//\\//\\
+
+  //\\//\\//\\ OUTLETS //\\//\\//\\
+  val out = AvroOutlet[CustomerTransaction]("transactions")
+
+  //\\//\\//\\ SHAPE //\\//\\//\\
+  final override val shape = StreamletShape(out)
+
+  //\\//\\//\\ LOGIC //\\//\\//\\
+  final override def createLogic = new RunnableGraphStreamletLogic {
+    val dataFrequency = FiniteDuration(streamletConfig.getInt("data-frequency"), "ms")
+
+    def runnableGraph =
+      GenerateTransactionsUtil.makeSource(dataFrequency)
+        .map(transaction ⇒ {
+          log.info("Reading Transaction: " + transaction.toString)
+          transaction
+        })
+        .to(plainSink(out))
+  }
+
+  val DataFrequency = IntegerConfigParameter(
+    key = "data-frequency",
+    description = "",
+    defaultValue = Some(1000)
+  )
+
+  override def configParameters = Vector(DataFrequency)
+
+}
+
+object GenerateTransactionsUtil {
+
+  val rootConfigKey = "fraud-detection"
+
+  lazy val dataFrequencyMilliseconds: FiniteDuration =
```

Comment:
I think this can be remove, not used anymore?

Reply:
removed!